swarm.sh

```diff
@@ -113,15 +100,17 @@ function initialize_package() {
 
     deploy_importers
 
-    restart_hapi_fhir
+    if [[ $DISABLE_VALIDATION == "false" ]]; then
+      restart_hapi_fhir
+    fi
   ) || {
     log error "Failed to deploy DISI package"
     exit 1
   }
 }
 
 function destroy_package() {
-  docker::service_destroy "${ALL_SERVICES[@]}"
+  docker::service_destroy "${SERVICE_NAMES[@]}"
```

Comment:
When destroying the config importers should be removed as well, not sure why you would remove it ?

Reply:
The config importers are removed soon after they have been run when we deploy (init/up command). We make use of the `docker::deploy_config_importer` function which removes the importers after they are run.

Reply:
Yeah, I think we have kept it in the destroy section in the past to ensure that if some weren't removed during the init/up commands, they get removed when we destroy, I think that is what Nour is referring to.